src/System.Runtime.Extensions/ref/project.json

```diff
@@ -1,6 +1,5 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.20"
```

Comment:
>"System.Runtime": "4.0.20" [](start = 4, length = 26)

I think you don't really need to remove this, since just having the P2P reference in the csproj will use that to compile instead of this one.